webpack.config.ts

```diff
@@ -242,6 +257,11 @@ module.exports = {
     new HtmlWebpackPlugin({
       chunks: ['exploration_player'],
       filename: 'exploration-player-page.mainpage.html',
+      meta: {
+        name: 'Personalized Online Learning from Oppia',
+        description: 'Oppia is a free, open-source learning platform. Join ' +
```

Comment:
If this value is same everywhere can't we make this as a constant (detault_meta)?